apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -174,7 +174,7 @@ export async function uploadAssessmentInstanceScores(
 
     // accumulate output lines in the "output" variable and actually
     // output put them in blocks, to avoid spamming the updates
-    let output: string | null = null;
+    let output = null as string | null;
```

Comment:
The type checking in typescript doesn't handle async code very well. This is a known issue and workaround.